src/main/features/claim/views/check-and-send.njk

```diff
@@ -26,9 +27,16 @@
     {{ csrfProtection(csrf) }}
 
     {{ tableStart("yourDetails", "Your details") }}
-    {{ row("Name:", claimantFullName) }}
-    {{ row("Date of birth:", dateOfBirth) }}
-    {{ row("Phone:", draftClaim.claimant.mobilePhone.number) }}
+    {{ row("About you:", claimantPartyType) }}
+    {{ row("Name:", name ) }}
+    {% if (partyType === 'COMPANY' or partyType === 'organisation') %}
```

Comment:
A `partyType ===  'COMPANY'` condition seem invalid, looks like it should be `partyType === 'company'`

Reply:
Fixed by @comanirudha